modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -50,17 +50,17 @@
         <template #bodyMain>
           <div v-if="docReady" class="apos-doc-editor__body">
             <AposSchema
-              v-for="(tab, index) in tabs"
+              v-for="tab in tabs"
               v-show="tab.name === currentTab"
-              :key="index"
-              :schema="tab.schema"
-              :current-fields="tab.fields"
+              :key="tab.name"
+              :schema="groups[tab.name].schema"
+              :current-fields="groups[tab.name].fields"
               :trigger-validation="triggerValidation"
               :utility-rail="false"
               :following-values="followingValues('other')"
               :doc-id="docId"
-              :value="docOtherFields"
-              @input="updateDocOtherFields"
+              :value="docFields"
```

Comment:
So we just pass all the data values to all the schemas but current-fields and the group schema subsets prevent them from abusing the  privilege. Yeah this is looking increasingly like the sane thing to do. 👍